drivers/sensor/max17262/max17262.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Matija Tudan
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_BATTERY_MAX17262_H_
+#define ZEPHYR_DRIVERS_SENSOR_BATTERY_MAX17262_H_
+
+/* Register addresses */
+enum {
+	STATUS          = 0x00,
+	REP_CAP         = 0x05,
+	REP_SOC         = 0x06,
+	INT_TEMP        = 0x08,
+	VCELL           = 0x09,
+	AVG_CURRENT     = 0x0b,
+	FULL_CAP_REP    = 0x10,
+	TTE             = 0x11,
+	CYCLES          = 0x17,
+	DESIGN_CAP      = 0x18,
+	ICHG_TERM       = 0x1E,
+	TTF             = 0x20,
+	VEMPTY          = 0x3A,
+	FSTAT           = 0x3D,
+	COULOMB_COUNTER = 0x4D,
+	SOFT_WAKEUP     = 0x60,
+	HIBCFG          = 0xBA,
+	MODELCFG        = 0xDB,
+};
+
+/* Masks */
+enum {
+	FSTAT_DNR        = 0x01,
+	STATUS_POR       = 0x02,
+	MODELCFG_REFRESH = 0x8000,
+};
+
+/* MAX17262 specific channels */
+enum max17262_channel {
+	MAX17262_COULOMB_COUNTER,
+};
+
+struct max17262_data {
+	const struct device *i2c;
```

Comment:
No need to put this in RAM now that we have DEVICE_DT_GET.

```suggestion
```

Reply:
Removed.